pytorch_lightning/loggers/base.py

```diff
@@ -335,6 +335,13 @@ def save_dir(self) -> Optional[str]:
         """
         return None
 
+    @property
+    def group_separator(self):
+        """
+        Return the default separator used by the logger to group the data into subfolders.
+        """
+        return '/'
```

Comment:
it this also used for the dir/file path?

Reply:
Right now it is used only for put the logs of the norms of the gradients in their specific subfolder. I'm not sure to which files or directory you are referring, but I think this can be also used to create more complex hierarchies also when storing file artifacts with a logger

Reply:
this for grouping logs, has nothing to do with file paths. 
the separator for file parts should be determined by the filesystem type and operating system.